glib/src/collections/strv.rs

```diff
@@ -1359,6 +1373,49 @@ impl<const N: usize> IntoStrV for [&'_ String; N] {
     }
 }
 
+// rustdoc-stripper-ignore-next
+/// Representation of a borrowed `NULL`-terminated C array of `NULL`-terminated UTF-8 strings.
+///
+/// It can be constructed safely from a `&StrV` and unsafely from a pointer to a C array.
+/// This type is very similar to `[GStringPtr]`, but with one added constraint: the underlying C array must be `NULL`-terminated.
+#[repr(transparent)]
+pub struct CStrV {
```

Comment:
The name seems inconsistent with everything else. This is basically the `String`/`PathBuf` (`StrV`) vs. `str`/`Path` (`CStrV`) duality. Maybe this one should've actually been `StrV` and the other one `StringV`. Using `Buf` here seems wrong.

In other places (e.g. in `gstreamer`) there's a pattern of `Foo` and `FooRef`, so maybe we can just call this one `StrVRef`? @bilelmoussaoui what do you think?

Also `GStringPtr` is an ugly name, should probably reconsider that before the next release. Its difference with `GStr` is that it doesn't store the length and is equivalent with the raw C pointer. `GStrLite` or so... doesn't have to be solved as part of this PR but in case someone has ideas while we talk about naming anyway.

Reply:
I'd still vote for `StrV` (stays as it is) and `StrVRef`. But waiting for @bilelmoussaoui to opine :)

Reply:
StrvRef / Strv sounds good to me